tests/Fixer/Alias/NoMixedEchoPrintFixerTest.php

```diff
@@ -151,42 +160,45 @@ public function testFixPrintToEcho(string $expected, ?string $input = null): voi
 
     public static function provideFixPrintToEchoCases(): iterable
     {
-        yield from [
-            [
-                '<?php
+        yield [
+            '<?php
                 echo "test";
                 ',
-            ],
-            [
-                '<?php
+        ];
+
+        yield [
+            '<?php
                 echo ("test");
                 ',
-            ],
-            [
-                '<?php
+        ];
+
+        yield [
+            '<?php
                 echo("test");
                 ',
-            ],
-            // https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/1502#issuecomment-156436229
-            [
-                '<?php
+        ];
+        // https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/1502#issuecomment-156436229
+        yield [
+            '<?php
                 ($some_var) ? print "true" : print "false";
                 ',
-            ],
-            // echo has no return value while print has a return value of 1 so it can be used in expressions.
-            // https://www.w3schools.com/php/php_echo_print.asp
-            [
-                '<?php
+        ];
+        // echo has no return value while print has a return value of 1 so it can be used in expressions.
```

Comment:
```suggestion
        ];

        // echo has no return value while print has a return value of 1 so it can be used in expressions.
```

looks like missing edge case for Fixer. maybe worth to handle it in fixer directly and regenerate this PR?